libraries/AP_Generator/AP_Generator_RichenPower.h

```diff
@@ -85,6 +84,7 @@ class AP_Generator_RichenPower
         RUN = 1,
         CHARGE = 2,
         BALANCE = 3,
+        OFF = 4,
```

Comment:
I probably would have made OFF the 0 item in the enum but.. doesn't really matter of course.